tests/unit/test_submit.py

```diff
@@ -42,6 +45,9 @@ def test_commit_files_parallel_change_to_upstream(tmp_path, workspace_repo):
     submit.git("clone", workspace_repo, checkout)
     (checkout / "project.yaml").write_text("project")
     (checkout / "codelist.csv").write_text("codelist")
+    code = checkout / "analysis/code"
+    code.parent.mkdir()
+    code.write_text("code")
```

Comment:
You probably know this, but you'll need to repeat this in the other_checkout

Reply:
:facepalm: